_layouts/post.html

```diff
@@ -4,17 +4,93 @@
 
 {%- assign locales = site.data.locales.post -%}
 <div class="blogholder img-responsive">
-  <article class="post" itemscope itemtype="http://schema.org/BlogPosting">
-	<div class="post-content" itemprop="articleBody">
-		{{ content }}
+	<article class="post" itemscope itemtype="http://schema.org/BlogPosting">
+		<div class="post-content" itemprop="articleBody">
+			{{ content }}
+		</div>
+	</article>
+	<aside>
 		<h2 id="{{ locales.connect[ page.lang ] | downcase | replace: ' ', '-' }}">{{ locales.connect[ page.lang ] }}</h2>
 		<ul>
 			<li>{{ locales.email[ page.lang ] }}: <a href="mailto:dto.btn@tbs-sct.gc.ca">dto.btn@tbs-sct.gc.ca</a></li>
 			<li>{{ locales.twitter[ page.lang ] }}: {{ locales.twitterhandle[ page.lang ] }}</li>
 			<li>{{ locales.slack[ page.lang ] }}: <a href="https://design-gc-conception.slack.com/join/shared_invite/enQtODE1OTc5Mzg5NzQ4LWQ3MjZjMTdjMjk2ZTZmMTJjYWQ3ZmRiNDYwYjRmN2NjYzQyNjFlNDBlY2FkNWE1ODg2YjExY2QwZmVjN2MwMGM">http://design-GC-conception.slack.com</a></li>
+			<li>{{ locales.signup[ page.lang ] }}:
+				<ul class="list-unstyled">
+					<li>
+						<form action="https://github.us12.list-manage.com/subscribe/post?u=5700d338d6ab413ebca1099f4&amp;id=c6bb0b9f64" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate paddingc" target="_blank" novalidate>
+							<div id="mc_embed_signup_scroll" class="well well-sm">
+								<fieldset class="gc-chckbxrdio">
+									<legend>{{ locales.emailtype[ page.lang ] }}</legend>
+									<ul class="list-unstyled lst-spcd-2">
+										<li class="radio">
+											<input type="radio" value="html" name="EMAILTYPE" id="mce-EMAILTYPE-0">
+											<label for="mce-EMAILTYPE-0">{{ locales.emailtype1[ page.lang ] }}</label>
+										</li>
+										<li class="radio">
+											<input type="radio" value="text" name="EMAILTYPE" id="mce-EMAILTYPE-1">
+											<label for="mce-EMAILTYPE-1">{{ locales.emailtype2[ page.lang ] }}</label> <small>{{ locales.emailtype-cmmt[ page.lang ] }}</small>
+										</li>
+									</ul>
+								</fieldset>
+								<div class="mc-field-group">
+									<div class="input-group">
+										<label class="wb-inv" for="mce-EMAIL">{{ locales.mce-email[ page.lang ] }}</label>
+										<input type="email" value="" name="EMAIL" class="form-control required email" id="mce-EMAIL" placeholder="{{ locales.mce-email[ page.lang ] }}">
+										<span class="input-group-btn">
+											<button type="submit" name="subscribe" id="mc-embedded-subscribe" class="btn btn-success nowrap">{{ locales.subscribe[ page.lang ] }}</button>
```

Comment:
@delisma Please make sure it still links to Mailchimp

Reply:
Yes the form does subscribe us in the DTO newsletter but the confirmation page has a button `contibue to our website` the leads to the wrong URL. The URL is https://canada-ca.github.io/ which brings people to a splash page that looks unstyled and still not bringing people to the right place.

![image](https://user-images.githubusercontent.com/2599251/121425791-cbd8b900-c940-11eb-89dc-32751bf81523.png)
![image](https://user-images.githubusercontent.com/2599251/121426137-3b4ea880-c941-11eb-8df4-0eab71781141.png)
These links go to https://canada-ca.github.io/blog-home-test-index.html and https://canada-ca.github.io/

Reply:
Made a [PR](https://github.com/canada-ca/canada-ca.github.io/pull/53) in github/canada-ca/canada.github.io repo, and changed the splash page.